app/views/providers/transactions/show.html.erb

```diff
@@ -20,15 +20,21 @@
           </th>
           <th data-sort-type="date" class="govuk-table__header sort header-sort-desc table-combine_right_if_narrow" scope="col">
             <span class="govuk-visually-hidden">Sort by </span>
-            <span class="aria-sort-description"><%= t('.col_date') %></span><span class="table-hidden_right_title"> &amp; <%= t('.col_description') %></span><span class="sort-direction-indicator"><span class="screen-reader-sort-indicator govuk-visually-hidden"> (currently sorted from newest to oldest)</span></span>
+            <span class="aria-sort-description"><%= t('.col_date') %></span><span class="table-hidden_right_title"> <%= t('.col_and_description') %></span><span class="sort-direction-indicator"><span class="screen-reader-sort-indicator govuk-visually-hidden"> (currently sorted from newest to oldest)</span></span>
```

Comment:
The text "currently sorted from newest to oldest" should be defined in a translation file - not here in the markup.

This code would be easier to read if the spans were split onto separate lines:
```html
<span class="aria-sort-description"><%= t('.col_date') %></span>
<span class="table-hidden_right_title"><%= t('.col_and_description') %></span>
<span class="sort-direction-indicator">
  <span class="screen-reader-sort-indicator govuk-visually-hidden"> (currently sorted from newest to oldest)</span>
</span>
```
Then sort out any spacing issues with CSS and not by compressing the markup.

Reply:
The arrangement of the `<span>` onto one line is to avoid unwanted whitespace.

Reply:
Why?

Reply:
The sorting arrows get pushed onto a new line on some narrow displays.